taxonomies/categories.txt

```diff
@@ -51758,23 +51855,23 @@ ciqual_food_code:en:25033
 ciqual_food_name:en:Burgundy-style beef stew
 ciqual_food_name:fr:Boeuf bourguignon
 
-<en:Beef dishes
+<en:Bœufs bourguignons
```

Comment:
There are 2 different things:
- "Boeufs bourguignons" (the prepared dish, with meat and vegetables)
- and "Beef stewing meat" ("Boeuf, à bourguignon ou pot-au-feu).

en:Boeufs bourguignons is under en:beef stews, and it should not be a parent of cooked or raw beef stew meat .

There are 2 AGB categories:

6231	6231	viandes, œufs, poissons	viandes crues	Boeuf, à bourguignon ou pot-au-feu, cru	Beef, stewing meat, raw
25033	25033	entrées et plats composés	plats composés	Boeuf bourguignon	Burgundy-style beef stew
11120	11120	aides culinaires et ingrédients divers	sauces	Sauce bourguignonne, préemballée	Burgundy-style sauce, prepacked

6231 | 6231 | viandes, œufs, poissons | viandes crues | Boeuf, à bourguignon ou pot-au-feu, cru | Beef, stewing meat, raw
-- | -- | -- | -- | -- | --
25033 | 25033 | entrées et plats composés | plats composés | Boeuf bourguignon | Burgundy-style beef stew
11120 | 11120 | aides culinaires et ingrédients divers | sauces | Sauce bourguignonne, préemballée | Burgundy-style sauce, prepacked



25033 | 25033 | entrées et plats composés | plats composés | Boeuf bourguignon | Burgundy-style beef stew
-- | -- | -- | -- | -- | --

Reply:
<en:Beef
en:Beef stew meat
fr:Boeuf à bourguignon, Boeuf à Pot-au-feu
agribalyse_food_code:en:6231
ciqual_food_code:en:6231
ciqual_food_name:en:Beef, stewing meat, raw
ciqual_food_name:fr:Boeuf, à bourguignon ou pot-au-feu, cru

Reply:
Sauce bourguignonne was already OK. Nothing to report.

Reply:
and Beef bourguignon is kept as a stew.